internal/template-bundle/bundle_test.go

```diff
@@ -1,48 +1,61 @@
 package template_bundle
 
 import (
+	"runtime"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+func retrieveSuffix() string {
+	switch runtime.GOARCH {
+	case "amd64":
+		return ""
+	case "s390x":
+		return "-s390x"
+	default:
+		return ""
+	}
+}
+
 var _ = Describe("Template bundle", Ordered, func() {
 	var (
 		testBundle Bundle
+		nameSuffix string
 	)
 
 	BeforeAll(func() {
 		var err error
 		testBundle, err = ReadBundle("template-bundle-test.yaml")
 		Expect(err).ToNot(HaveOccurred())
+		nameSuffix = retrieveSuffix()
 	})
 
 	It("should correctly read templates", func() {
```

Comment:
This test should iterate over all supported archs, independent of the hosts architecture. Consider making it a table.

Reply:
The is that the templates for this provisional solution are read in the readTemplates function based on the `runtime.GOARCH` constant. While I could attempt to mock this, it would come at the cost of readability.

If the common templates release files for different architectures, we can revert these changes and implement the necessary modifications in the `CreateControllers` function within `setup.go`. What are your thoughts on this?

Reply:
Done.